### ISSUE TITLE
Als gemeente wil ik een api-specificatie voor het bevragen, toevoegen en wijzigen van producten

### ISSUE THREAD

**Issue by frankmijn** [VNG-Realisatie/producttypecatalogus#32]
zodat ik deze kan gebruiken in mijn projecten.

**Definition of done**
- [ ] besluit relevant/generiek
- [ ] indien relevant/generiek; besluit verwerken in kernmodel of extensie
- [ ] indien kernmodel; verwerkt in model

**Comment by frankmijn** [comment]
Ja, ben ik met je eens @joeribekker. Zouden we de user stories die we hebben ingediend waarop we het informatiemodel aanpassen/uitbreiden kunnen gebruiken als basis? Want deze waren uiteindelijk ingestoken op wat je aan de voorkant zou willen ontsluiten.

**Comment by joeribekker** [comment]
Kan, maar komt een beetje meer op "ik wil crud actors kunnen uitvoeren". Kan hoor, maar wellicht komen specifiekere use cases tot een betere structuur (wat wel/niet inline ontsluiten aan informatie, wat mag je wel/niet updaten)

**Comment by frankmijn** [comment]
@joeribekker snap wat je bedoelt, dan zouden we de use cases echt moeten insteken vanuit functionaliteiten die we in laag 4/5 ter beschikking stellen. Dat zijn inderdaad specifiekere use cases.
